cluster/validation.go

```diff
@@ -672,6 +677,29 @@ func validatePSP(c *Cluster) error {
 	return nil
 }
 
+func validatePodSecurity(c *Cluster) error {
+	parsedVersion, err := getClusterVersion(c.Version)
+	if err != nil {
+		logrus.Warnf("Failed to parse semver range for validating Pod Security")
+		return err
+	}
+	logrus.Debugf("Checking PodSecurity for cluster version [%s]", c.Version)
+	level := c.Services.KubeAPI.PodSecurityConfiguration
+	if len(level) != 0 {
```

Comment:
Could you add a comment saying why checking the level of the PodSecurityConfiguration is important for those who are less familiar with PSP?

Reply:
sure, comment is added